src/store.cc

```diff
@@ -1208,7 +1208,7 @@ storeLateRelease(void *)
     StoreEntry *e;
     static int n = 0;
 
-    if (Store::Controller::store_dirs_rebuilding) {
+    if (!Store::Root().indexReady()) {
```

Comment:
Is it possible for Store::Root() to assert when called in this context because Store::FreeMemory() has been called already due to an ongoing shutdown that started while we were waiting for the storeLateRelease event?